motoko/encrypted-notes-dapp/README.md

```diff
@@ -104,8 +102,6 @@ Once authenticated with II:
 
 ### Adding a new device
```

Comment:
Nothing is listed under Adding a new device header. Should we remove this header then?

Reply:
Good catch, thanks!